--- desmume-0.8/src/gtk/desmume.desktop.in~	2008-04-27 13:29:12.558448196 +0200
+++ desmume-0.8/src/gtk/desmume.desktop.in	2008-04-27 13:31:21.697955134 +0200
@@ -2,9 +2,11 @@
 Encoding=UTF-8
 Version=@VERSION@
 Type=Application
-Name=DeSmuME (Gtk)
+Name=DeSmuME
 Comment=Nintento DS emulator
+Comment[pl]=Emulator Nintendo DS
 TryExec=desmume
 Exec=desmume
 Icon=DeSmuME.xpm
-Categories=GNOME;GTK;Game;Emulator
+Categories=Application;Game;
+# vi: encoding=utf-8
